### PR TITLE
Set --available option as a default for dnf repoquery

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: doc/repoquery.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
     Limit the resulting set to leaves packages that were installed as dependencies so they are no longer needed. This switch lists packages that are going to be removed after executing ``dnf autoremove`` command.
 
 ``--available``
-    Limit the resulting set to available packages only.
+    Limit the resulting set to available packages only (set by default).
```
Comment (ignatenkobrain) [inline doc/repoquery.rst]: shouldn't we say that it's deprecated?

**Comment by ignatenkobrain** [comment]
what about `--installed` option?

**Comment by j-mracek** [comment]
--installed works like expected, same as --available. But you know, there can be only one default value :-)  .

**Comment by j-mracek** [comment]
I am going to allow combination of --installed and --available options

File: doc/repoquery.rst (hunk 1 above)
Comment (j-mracek) [inline doc/repoquery.rst]: and what about only (default) ? I am enabling of --installed with --available, therefore it is still handy.

**Comment by ignatenkobrain** [comment]
I don't think that it's good idea

**Comment by j-mracek** [comment]
@ignatenkobrain Why? - It was default behavior of repoquery (installed + available) and may be someone like it. Now we change default to available, but it will be nice to provide an option combination that can mimic formal behavior.

**Comment by j-mracek** [comment]
Jenkins, test this please

**Comment by mluscon** [comment]
again, incompatible change -> dnf-2.0

**Pull request closed without merging**